DEVELOPMENT.md

```diff
@@ -151,6 +151,84 @@ to start a debugger to which attach from your IDE.
    curl localhost:8081/apis/apps/v1/deployments
    ```
 
+## Database
+
+### Database requirements
+
+* psql
+```bash
+$ sudo dnf install postgresql
+$ psql --version
+psql (PostgreSQL) 15.6
+```
+* A SQL client is recommended to interact easily with the database. For example [DBeaver Community](https://dbeaver.io/).
+
+### Create a Table and Populate the Database
+
+The script `database/init_db.go` will create a table in the database and will insert the test objects from `database/test_objects.sql`
+* Add the database access configuration. The values are the same from the `ps-secret.yaml` or the ones
+  that you have defined in the `values.yaml`.
+```go
+const (
+	host     = "localhost"
+	port     = 5432
+	user     = // the POSTGRES_USER from ps-secret.yaml, ie: "admin", "ps_user".
+	password = // the POSTGRES_PASSWORD from ps-secret.yaml
+	dbname   = // the POSTGRES_DB from ps-secret.yaml, ie: "postgres", "test_db".
```

Comment:
If these values have defaults on the chart values file, they need to have defaults here.

Reply:
I modified these values to have the same as the defaults, but the lint is failing with `Potential hardcoded credentials (gosec)`, which is true.

Reply:
fixed now using flags.